### PR TITLE
HealthCheck container

### PR DESCRIPTION
**Pull request by sah4ez**
Added NAME highlight depending on container health status. For healthy -  green, starting - yellow, unhealthy - magenta.

### CODE REVIEW

File: cwidgets/compact/text.go [hunk 1]
```diff
@@ -33,3 +40,19 @@ func (w *TextCol) Reset() {
...
 	w.Text = s
 }
+
+func (w *TextCol) Color(s string){
+	color := ui.ThemeAttr("par.text.fg")
+	if w.isHighlight{
+		color = ui.ThemeAttr("par.text.hi")
+	}
+	switch s {
+	case "health":
```
Comment (bcicen) [inline cwidgets/compact/text.go]: should this be `healthy`?

Comment (sah4ez) [reply cwidgets/compact/text.go]: Ohh..., yes. I made a typo.
Need up new pull-request?

File: cwidgets/compact/status.go [hunk 2]
```diff
@@ -35,6 +35,10 @@ func (s *Status) Set(val string) {
 		color = ui.ColorGreen
 	case "exited":
 		color = ui.ColorRed
+	case "unhealthy":
```
Comment (bcicen) [inline cwidgets/compact/status.go]: doesn't look like these changes were used anywhere

**Comment by bcicen** [comment]
Looks good @sah4ez -- if you can remove the unused changes `cwidgets/compact/status.go` and squash your commits, I'll get this merged.

**Comment by sah4ez** [comment]
I removed unused colours from `cwidgets/compact/status.go`

**Comment by bcicen** [comment]
merged -- fyi, I removed the need for duplicate container inspection in https://github.com/bcicen/ctop/commit/a1ebf3f90edd1a6bb9e137acb2f42d6ef5be8423 by listening for the `health_status` docker event

**Pull request merged**